db/v-2/tables/vulnerability-metadata.xml

```diff
@@ -15,4 +15,8 @@
         <modifyDataType tableName="vulnerability_metadata" columnName="severity_num" newDataType="float"/>
     </changeSet>
 
+    <changeSet id="vulnerability-metadata-3" author="frolov">
+        <modifyDataType tableName="vulnerability_metadata" columnName="severity_num" newDataType="DECIMAL(2,1)"/>
```

Comment:
what's about changing the existed content?

Reply:
It will be removed from the database :)